lms/templates/shoppingcart/shopping_cart.html

```diff
@@ -141,14 +144,32 @@
         <a href="${marketing_link('COURSES')}" class="blue">${_('View Courses')}</a>
       </div>
     % endif
-
 </div>
+<form id='refresh_tracker'>
+   <input id="reloadValue" type="hidden" name="reloadValue" value="" />
+</form>
 </%block>
 <script>
     $(function() {
         var isSpinnerBtnEnabled = true;
         var prevQty = 0;
 
+        // This snipnet of code is to detect when and
+        // end-user has clicked the back button to the
+        // page. We want to force refresh to make
+        // sure there is no cached data happening
+        var d = new Date();
+        d = d.getTime();
+        if (jQuery('#reloadValue').val().length == 0)
+        {
+          jQuery('#reloadValue').val(d);
+        } else
+        {
+          jQuery('body').hide();
+          jQuery('#reloadValue').val('');
+          location.reload();
```

Comment:
change it to location.reload(true) This reloads the current page without using the cache. 

location.reload() takes default value to false and it reloads the current page from the cache.